usim/__about__.py

```diff
@@ -10,10 +10,10 @@
 
 .. code:: python3
 
-   >>> from usim import each, run
+   >>> from usim import delay, run
    >>>
    >>> async def metronome(period: float, sound: str):
-   ...     async for now in each(delay=period):
+   ...     async for now in delay(value=period):
```

Comment:
See above comment. (I'd recommend pulling the examples from a single source but have no idea how to do that.)

Reply:
See #45.